meerkat_frontend/src/js/technical/tableManager.js

```diff
@@ -334,8 +334,17 @@ function callTableOptionButton(element, redrawFunctionName) {
     :param object variables:
         An object containing details for given variable IDs, as returned by Meerkat API `/variables`.
         Specifically used to print the variable name instead of ID.
+    :param object alerts_table_config
+        Set of settings to customise the table
  */
-function drawAlertsTable(containerID, alerts, variables) {
+function drawAlertsTable(containerID, alerts, variables, alerts_table_config) {
+
+    var district_column = false;
+    if(alerts_table_config === undefined ){
+        district_column = false; //default
+    }else{
+        district_column = alerts_table_config.district_column;
+    }
```

Comment:
I think I'd prefer to condense these 6 lines down to the following 2 lines:
`alerts_table_config = alerts_table_config || {};`
`alerts_table_config.district_column = alerts_table_config.district_column || false;`
and you would likewise have one new line for every configuration that needs a default value...
To me it's more readable; each line fulfills a distinct single job (setting a default value), so the code has a clearer structure.